src/airtableApi.jsx

```diff
@@ -0,0 +1,77 @@
+import {
+  AIRTABLE_TABLE_NAME,
+  AIRTABLE_BASE_ID,
+  AIRTABLE_API_KEY,
+} from './configuration.jsx';
+
+const tableUrl = `https://api.airtable.com/v0/${AIRTABLE_BASE_ID}/${AIRTABLE_TABLE_NAME}`;
```

Comment:
defining the url and the default headers at the top means I don't have to repeat the same code.